### DISCUSSION TITLE
Sidekick64 with C16.... problems with SID emulation

### DISCUSSION THREAD

**Discussion by gms0012** [frntc/Sidekick64#59] [General]
I am using Sidekick64 with my C16...

but when I activate the SID emulation, I can not load anything. The system always freezes. 

When I dont activate the SID emulation I can load the "demo sound prgs"... but of course without sound... :) 

please help me....

![111](https://user-images.githubusercontent.com/1883546/222229711-7db59645-2879-4e44-ac32-8b8251e4eddf.png)

**Comment by frntc** [comment]
If never experienced or heard of this problem, but let's see... which RPi model are you using? Do you have a good power supply? Can you go to basic and try a program that uses the SID for testing?

**Comment by gms0012** [comment]
i have a RPi 3 Model B V2 and an power supply with 2500 mA

Do I have to make the correct SID settings that a can program load?  When using the default settings, the default programs crash when loading (changed only output to JACK)

yes, i can try a basic program.... which program do u want to test me


also.. when I start FB16.. i dont see any files... only a folder CD? 


By the way... do I also have to turn on the C16? Its normalls gets power from the RPI3+.

thanks for your help

**Comment by frntc** [comment]
First of all: it seems you closed the "power" jumper on the Sidekick -- you must NOT do this with a C16. Open it, and power C16 and RPi separately! (otherwise you power the entire C16 via your RPi and this might cause damage)

If your RPi is a "B" and not "B+" it will not work. Only "3A+" and "3B+" will do the job.

When you launch FB16 it only sees local drives, not the Sidekick-SD card.

**Reply by gms0012** [reply]
thanks for the reply,,, I was pretty sure that I have a 3+.. but your question forced me to look at it again.. and its no + modell.. so my bad..

i will try to get a B+...

and then i will remove the power jumper. 

any chance that u will provide a solution for RASP 4?

Thanks

Gregor

**Comment by frntc** [comment]
The RPi 4 is a different SoC. Some basic functionality did work, but getting all the timings and caching right is challenging -- in short, no plans right now.
